docs/tutorials/demo_data_science_project.ipynb

```diff
@@ -131,8 +129,7 @@
    "source": [
     "from lakefs_spec.client_helpers import create_repository\n",
     "\n",
-    "create_repository(client=fs.client, name=REPO_NAME,\n",
-    "                  storage_namespace=f\"local://{REPO_NAME}\")"
+    "repo = create_repository(client=fs.client, name=REPO_NAME, storage_namespace=f\"local://{REPO_NAME}\")"
```

Comment:
After executing this part of code I got:
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[2], line 1
----> 1 from lakefs_spec.client_helpers import create_repository
      3 repo = create_repository(client=fs.client, name=REPO_NAME, storage_namespace=f"[local://{](local://{REPO_NAME)[REPO_NAME](local://{REPO_NAME)}")

ImportError: cannot import name 'create_repository' from 'lakefs_spec.client_helpers' (C:\Users\Maciej Janiszewski\.venv\Lib\site-packages\lakefs_spec\client_helpers.py)